briskedu/brisk-learning
# Add Sentry integration

Summary:
Adds integration for Sentry so that we can be informed of any errors that arise.  This should be better than Datadog for catching low-volume issues.  I'll just configure it to email me for every error for now.

https://app.asana.com/0/1203963408726580/1204099843417540

Test Plan:
1. Confirm user data is passed back when I remove the production check
2. Confirm it has email and ignored
3. Confirm environment says development
